commands/remove.go

```diff
@@ -44,7 +44,7 @@ func runDelete(cmd *cobra.Command, args []string) error {
 	var services stack.Services
 	var gatewayAddress string
 	var yamlGateway string
-	if len(yamlFile) > 0 {
+	if len(yamlFile) > 0 && len(args) == 0 {
```

Comment:
I think this is the only command we need to change, but it will require a unit test too.

If you're not keen on doing the work, or it's a bit more than you wanted to do, then please close the PR and we'll prioritise the issue accordingly.